apps/zadmin/tests/test_tasks.py

```diff
@@ -86,102 +95,101 @@ def fetch_langpacks(self, version):
+        assert self.get_langpacks().count() == 1
 
         self.fetch_langpacks(versions[1])
 
         langpacks = self.get_langpacks()
-        eq_(langpacks.count(), 1)
+        assert langpacks.count() == 1
 
         a = langpacks[0]
-        eq_(a.versions.count(), 2)
+        assert a.versions.count() == 2
 
         v = a.versions.get(version=versions[1])
-        eq_(v.files.all()[0].status, amo.STATUS_PUBLIC)
+        assert v.files.all()[0].status == amo.STATUS_PUBLIC
 
     def test_fetch_duplicate_langpack(self):
         self.fetch_langpacks(amo.FIREFOX.latest_version)
 
         langpacks = self.get_langpacks()
-        eq_(langpacks.count(), 1)
-        eq_(langpacks[0].versions.count(), 1)
-        eq_(langpacks[0].versions.all()[0].version,
-            amo.FIREFOX.latest_version)
+        assert langpacks.count() == 1
+        assert langpacks[0].versions.count() == 1
+        assert (langpacks[0].versions.all()[0].version ==
+                amo.FIREFOX.latest_version)
 
         self.fetch_langpacks(amo.FIREFOX.latest_version)
 
         langpacks = self.get_langpacks()
-        eq_(langpacks.count(), 1)
-        eq_(langpacks[0].versions.count(), 1)
-        eq_(langpacks[0].versions.all()[0].version,
-            amo.FIREFOX.latest_version)
+        assert langpacks.count() == 1
+        assert langpacks[0].versions.count() == 1
+        assert (langpacks[0].versions.all()[0].version ==
+                amo.FIREFOX.latest_version)
 
     def test_fetch_updated_langpack_beta(self):
         versions = ('16.0', '16.0a2')
 
         self.fetch_langpacks(versions[0])
 
-        eq_(self.get_langpacks().count(), 1)
+        assert self.get_langpacks().count() == 1
 
         self.fetch_langpacks(versions[1])
 
         langpacks = self.get_langpacks()
-        eq_(langpacks.count(), 1)
+        assert langpacks.count() == 1
 
         a = langpacks[0]
-        eq_(a.versions.count(), 2)
+        assert a.versions.count() == 2
 
         v = a.versions.get(version=versions[1])
-        eq_(v.files.all()[0].status, amo.STATUS_BETA)
+        assert v.files.all()[0].status == amo.STATUS_BETA
 
     def test_fetch_new_langpack_beta(self):
         self.fetch_langpacks('16.0a2')
 
-        eq_(self.get_langpacks().count(), 0)
+        assert self.get_langpacks().count() == 0
 
     def test_fetch_langpack_wrong_owner(self):
         Addon.objects.create(guid='langpack-de-DE@firefox.mozilla.org',
                              type=amo.ADDON_LPAPP)
 
         self.fetch_langpacks(amo.FIREFOX.latest_version)
-        eq_(self.get_langpacks().count(), 0)
+        assert self.get_langpacks().count() == 0
 
     def test_fetch_langpack_invalid_path_fails(self):
         self.mock_request.return_value = None
 
         with self.assertRaises(ValueError) as exc:
             tasks.fetch_langpacks('../foo/')
-        eq_(str(exc.exception), 'Invalid path')
+        assert str(exc.exception) == 'Invalid path'
```

Comment:
and final bonus points for switching a whole test file to use `assert`s :bouquet: